bowling-ball/Game.cs

```diff
@@ -2,16 +2,63 @@
 
 namespace BowlingBall
 {
+    
     public class Game
     {
-        public void Roll(int pins)
-        {
-            throw new NotImplementedException();
+        private int[] Rolls = new int[21];
+        private int currentRoll;
+        public void Roll(int pins)
+        {
+            Rolls[currentRoll++] = pins;
         }
 
-        public int GetScore()
+        public int GetScore()
+        {
+            int score = 0;
+            int roll = 0;

```

Comment:
Rename this to rollIndex, as this is used to access specific elements of rolls collection